Deeploy/CommonExtensions/TypeCheckers/SignPropTypeChecker.py

```
@@ -45,9 +47,14 @@ def typeInferOutput(self, ctxt: NetworkContext, node: gs.Node,
         if signProp:
             nLevels = self._inferNumLevels(inputs, operatorRepresentation)
             signedness = self._inferSignedness(inputs, operatorRepresentation)
-
-            for obj, nLevels, sign in zip(outputs, nLevels, signedness):
-                obj.nLevels = nLevels
+            for obj, nLevel, sign in zip(outputs, nLevels, signedness):
```

Ruff (0.12.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)